content/posts/blog/2021-01-20.md

```diff
@@ -0,0 +1,55 @@
+
+Actually, most will just add another messenger to their list of messengers. Likely, that is more frustrating than helping, which brings me back to the beginning and the actual question: What issue or what problem does one want to solve here? What are the incentives? Can one decide towards a solution that, when the next hype chases through the internet, will have a better basis?
+
+A possible background of that question may be different expectations of some functionality, the basic requirement of effective data privacy and sovereignty or simply the possibility to reach all my contacts. However, that change will still require leaving behind the previous messaging system and switching to a new one, which almost always results in the loss of the existing contact connections. Furthermore, in terms of privacy, one is almost always required to provide a phone number. Again, what does one will try to solve?
+
+### Sovereignty of Communication
+
+So, as you likely already wondered, why am I going to advocate for XMPP now? I think the choice one should do firsthand, is not a choice of the messenger you use, but of the underlying software technology. Obviously, one should actually always do that. So, here we are: Consider the choice of technology first before you jump from one recommendation to the next one.
+
+> XMPP is an open protocol, as HTTP is for websites. It mostly does not matter how your website looks like, but everyone can interact with it. This principle is the idea behind XMPP for instant messaging.
+
+We at the XMPP Standards Foundation, but also many others involved in this open technology, think that XMPP is a good choice as a communication technology when it comes to this question. XMPP as a protocol has been around for about two decades since its creation in 1999 and a lot of experience has been gathered during that period. Mainly, this includes several important key indicators for the choice of technology and enabling of sovereignty as 
+there are:
+
+- Decentralization (federation) of communication services
+- Standardization and extensibility of technology
+- Innovation and the use of open development
+- Privacy and control, also by trustful encryption
+- Enabling of free speech and freedom of software
+
+> XMPP provides defined information how to handle the communication in a network. The user decides which messaging software suits or where the data should be stored.
+
+In contrast to the current situation of many people suggesting one specific messenger, in the XMPP universe you have the choice between many messengers – pick the one you like and not what others like (and maybe think that you might like as well) until the next hype starts. The difference is that you still have the comfort to be in the same network, independent of your choice today. I think this is a real solution.
```

Comment:
"you have **a** choice between many messengers"

"until the next internet trend", maybe.

Reply:
yes!